### PR TITLE
Edit button to select in pal

### PR DESCRIPTION
**Pull request by howard-stearns**
Adds a "people" button to the edit.js entity list.  When an object is selected in the editor, and you push that button, it will open the PAL and select the user who last edited that object, if they are still present in your session.

### CODE REVIEW

**Comment by howard-stearns** [comment]
test plan:
Run pal.js from this PR.
With two users in a domain, have one of them edit an object.
- The other user selects the object using edit.js, and clicks the people button in the entity list. (It is between "Jump to selection" and "Delete".) This opens the People Action List (if it is not already open), and selects that user. This means that:
 - The user is highlighted within the People Action List window.
 - The user is highlighted (with a red transparent ball) in the scene.
 - All of the objects that were last edited by this user in this session are highlighted (orange wireframe boun ding box), including the one that was selected in edit.js.
- If no object is selected, edit.js tells you with a modal alert.
- If the user is not in your People Action List (because they have left or have ignored you, which looks to you like they have left), the People Action List is still opened, but it tells you that user is not present.
- The People Action List does not currently support the simultaneous selection of multiple users. If multiple objects are selected in edit.js such that multiple users would need to be selected, the People Action List is still opened, but it tells you that multiple users would need to be selected.

Note: Some entities have entity scripts which everyone runs, such that everyone edits the object. When the People Action List shows all the objects last edited by someone, it may include such entities.

**Comment by hifi-gustavo** [comment]
Build console output for commit: ef02c84759e789fe5c51dfdbb2c83c857e670908 available at: https://gist.github.com/e3bcad53a751e0709f6199085dca55f0<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9319-ef02c84.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9319-ef02c84.dmg

**Comment by Nex-Pro** [comment]
Tested on 3x Win 10 desktops. geographically separate avatars.

### PASS.

With two users in a domain, have one of them edit an object. **CHECK**

The other user selects the object using edit.js, and clicks the people button in the entity list. (It is between "Jump to selection" and "Delete".) **CHECK**

This opens the People Action List (if it is not already open), and selects that user. **CHECK**

This means that:
The user is highlighted within the People Action List window. **PASS**
The user is highlighted (with a red transparent ball) in the scene. **PASS**

All of the objects that were last edited by this user in this session are highlighted (orange wireframe bounding box), including the one that was selected in edit.js. **PASS**

If no object is selected, edit.js tells you with a modal alert. **PASS**

If the user is not in your People Action List (because they have left or have ignored you, which looks to you like they have left), the People Action List is still opened, but it tells you that user is not present. **PASS**

The People Action List does not currently support the simultaneous selection of multiple users. If multiple objects are selected in edit.js such that multiple users would need to be selected, the People Action List is still opened, but it tells you that multiple users would need to be selected. ***"Only one user can be selected at a time."*** **is displayed on the PAL.**

Works at distance also.
Note:
"There were no recent users of the 0 selected objects" Could this say 'No objects selected' ?

**Comment by howard-stearns** [comment]
Thanks @Nex-Pro. Yeah, I think I'll change that wording.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 83685740c477a02776037701fa0f2989d0d47f90 available at: https://gist.github.com/e3bcad53a751e0709f6199085dca55f0<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9319-8368574.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9319-8368574.dmg

**Comment by howard-stearns** [comment]
https://github.com/highfidelity/hifi/pull/9319

File: scripts/system/libraries/entityList.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ var ENTITY_LIST_HTML_URL = Script.resolvePath('../html/entityList.html');
 EntityListTool = function(opts) {
     var that = {};
 
+
```
Comment (davidkelly) [inline scripts/system/libraries/entityList.js]: extra line here?

File: interface/resources/qml/hifi/LetterboxMessage.qml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Item {
+    property alias text: popupText.text
+    visible: false
+    id: letterbox
+    anchors.fill: parent
+    Rectangle {
+        anchors.fill: parent
+        color: "black"
+        opacity: 0.5
+        radius: hifi.dimensions.borderRadius
```
Comment (zfox23) [inline interface/resources/qml/hifi/LetterboxMessage.qml]: I think this should be a property of the LetterboxMessage. I don't think the radius of the background rectangle will always be this value.

File: interface/resources/qml/hifi/Pal.qml [hunk 3]
```diff
@@ -349,7 +339,9 @@ Item {
             anchors.fill: parent
             acceptedButtons: Qt.LeftButton
             hoverEnabled: true
-            onClicked: namesPopup.visible = true
+            onClicked: letterbox("Bold names in the list are Avatar Display Names.\n" +
```
Comment (zfox23) [inline interface/resources/qml/hifi/Pal.qml]: This is wonderful. So much better.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2f5f53c8e4c2c44b71d0fc3bbc347d74852eab5c available at: https://gist.github.com/e3bcad53a751e0709f6199085dca55f0<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9319-2f5f53c.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9319-2f5f53c.dmg

**Pull request merged**